src/main/java/org/cactoos/map/MapOf.java

```diff
@@ -106,8 +104,10 @@ public MapOf(final Map<X, Y> src, final Map.Entry<X, Y>... list) {
      * @param <Z> Type of items in the list
      * @since 0.11
      */
-    public <Z> MapOf(final Func<Z, Map.Entry<X, Y>> entry,
-        final Iterable<Z> list) {
+    public <Z> MapOf(
+        final Func<Z, Map.Entry<? extends X, ? extends Y>> entry,
```

Comment:
@andreoss I believe the func should be taking a `? super Z` no?

Reply:
@victornoel Done.